### PR TITLE
TPO Screen PR

### PR DESCRIPTION
**Pull request by harsh-mody**
@SirusCodes

### CODE REVIEW

File: assets/data/companys.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[
```
Comment (SirusCodes) [inline assets/data/companys.json:1]: companies.json
spelling mistake ;)

Comment (SirusCodes) [reply assets/data/companys.json:1]: Also I guess you are missing few companies

File: lib/screens/committees_screen.dart [hunk 2]
```diff
@@ -45,9 +43,8 @@ class _CommitteesScreenState extends State<CommitteesScreen> {
     return CustomScaffold(
       body: ListView(
         children: <Widget>[
-          CustomAppBar(
+          const CustomAppBar(
             title: "Committees & Events",
-            image: Image.asset(ImageAssets.committes),
```
Comment (SirusCodes) [inline lib/screens/committees_screen.dart:50]: Why you removed the image?

File: lib/screens/department_screen/widgets/faculty_details_section.dart [hunk 3]
```diff
@@ -35,7 +33,8 @@ class _FacultyDetailsSectionState extends State<FacultyDetailsSection> {
 
     deptFileName = deptMap[widget.department]!;
 
-    final data = await rootBundle.loadString("assets/data/faculty_details/$deptFileName.json");
+    final data = await rootBundle
+        .loadString("assets/data/faculty_details/$deptFileName.json");
```
Comment (SirusCodes) [inline lib/screens/department_screen/widgets/faculty_details_section.dart:37]: Pull

File: lib/screens/notification_screen/notification_screen.dart [hunk 4]
```diff
@@ -31,10 +30,9 @@ class _NotificationScreenState extends State<NotificationScreen> {
     return CustomScaffold(
       body: NestedScrollView(
         headerSliverBuilder: (context, innerBoxIsScrolled) => [
-          SliverToBoxAdapter(
+          const SliverToBoxAdapter(
             child: CustomAppBar(
               title: "Notifications",
-              image: Image.asset(ImageAssets.committes),
```
Comment (SirusCodes) [inline lib/screens/notification_screen/notification_screen.dart:37]: Again you removed the image

File: lib/screens/tpo_screen.dart [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+    _companys = _getCompanys();
+  }
+
+  List<Widget> getCompanyCards(List<CompanyModel> _companys) {
+    List<Widget> list = [];
+    for (var i = 0; i < _companys.length; i++) {
+      list.add(
+        Padding(
+          padding: const EdgeInsets.only(
+              top: 10, bottom: 10, left: 1.25, right: 1.25),
```
Comment (SirusCodes) [inline lib/screens/tpo_screen.dart:36]: Add comma

File: lib/screens/tpo_screen.dart [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+                    children: getCompanyCards(data),
+                  ),
+                );
+              }
+              return const Center(
+                child: CircularProgressIndicator(),
+              );
+            },
+          ),
+        ],
```
Comment (SirusCodes) [inline lib/screens/tpo_screen.dart:109]: You can use NestedScrollView

File: lib/utils/image_assets.dart [hunk 7]
```diff
@@ -24,6 +25,15 @@ class ImageAssets {
...
   static const codecell = "$_committeeLogo/codecell.png";
 
+  // company logos
+  static const _companyLogo = "$_root/company_logo";
+  static const accenture = "$_companyLogo/accenture.png";
+  static const deutsche_bank = "$_companyLogo/deutsche_bank.png";
+  static const infosys = "$_companyLogo/infosys.png";
+  static const jp_morgan = "$_companyLogo/jp_morgan.png";
+  static const oracle = "$_companyLogo/oracle.png";
+  static const tcs = "$_companyLogo/tcs.png";
```
Comment (SirusCodes) [inline lib/utils/image_assets.dart:35]: We dont need this...

File: lib/widgets/custom_app_bar.dart [hunk 8]
```diff
@@ -8,11 +9,24 @@ class CustomAppBar extends ConsumerWidget with PreferredSizeWidget {
...
+    } else if (title == "Training & Placement Cell") {
+      return Image.asset(
+        ImageAssets.tpo,
+        width: 177,
+      );
+    } else if (title == "") {
+      return Image.asset(ImageAssets.committes, width: 177);
+    }
+    return const SizedBox(width: 177);
+  }
```
Comment (SirusCodes) [inline lib/widgets/custom_app_bar.dart:29]: Remove this, previous implementation felt better

File: macos/Flutter/GeneratedPluginRegistrant.swift [hunk 9]
```diff
@@ -5,14 +5,30 @@
 import FlutterMacOS
```
Comment (SirusCodes) [inline macos/Flutter/GeneratedPluginRegistrant.swift:5]: Add this to .gitignore

**Review by SirusCodes — CHANGES_REQUESTED** [review]

File: assets/data/companys.json (hunk 1 above)
Comment (harsh-mody) [reply assets/data/companys.json:1]: Companies present in UI MockUps are all added
Spelling issue is fixed

File: lib/screens/notification_screen/notification_screen.dart (hunk 4 above)
Comment (harsh-mody) [reply lib/screens/notification_screen/notification_screen.dart:37]: Ha its handled in a different way just

File: lib/screens/committees_screen.dart (hunk 2 above)
Comment (harsh-mody) [reply lib/screens/committees_screen.dart:50]: Its handled in class as I needed custom specs for my image

File: lib/screens/department_screen/widgets/faculty_details_section.dart (hunk 3 above)
Comment (harsh-mody) [reply lib/screens/department_screen/widgets/faculty_details_section.dart:37]: Kiya ton hai?

File: lib/screens/tpo_screen.dart (hunk 6 above)
Comment (harsh-mody) [reply lib/screens/tpo_screen.dart:109]: Yes but Let it be now

File: lib/utils/image_assets.dart (hunk 7 above)
Comment (harsh-mody) [reply lib/utils/image_assets.dart:35]: Aah yes, I removed earlier I forgot ki we will use Firebase

File: lib/widgets/custom_app_bar.dart (hunk 8 above)
Comment (harsh-mody) [reply lib/widgets/custom_app_bar.dart:29]: Yeah but in this way, the widget can be customised as per app bar. We can do it for weird ones like Login also

File: macos/Flutter/GeneratedPluginRegistrant.swift (hunk 9 above)
Comment (harsh-mody) [reply macos/Flutter/GeneratedPluginRegistrant.swift:5]: Cool

File: lib/widgets/custom_app_bar.dart (hunk 8 above)
Comment (harsh-mody) [reply lib/widgets/custom_app_bar.dart:29]: Anyways will fix this

File: assets/data/companies.json [hunk 10]
```diff
@@ -1,26 +1,163 @@
...
-    },
-    {
-        "name": "TCS",
-        "image": "https://firebasestorage.googleapis.com/v0/b/tsec-app.appspot.com/o/company%2Ftcs.png?alt=media&token=0efc6904-6ec1-4086-8f40-8ba0275bb0b2"
-    }
-]
\ No newline at end of file
+[ {
+    "name" : "ACCENTURE",
+    "image" : "https://firebasestorage.googleapis.com/v0/b/tsec-app.appspot.com/o/Recruiters%2Faccenture.jpg?alt=media&token=b0d932e1-82de-4a92-910f-3b3947366859\\\""
```
Comment (SirusCodes) [inline assets/data/companies.json:3]: remove \\\ at end
And format it

**Review by SirusCodes — COMMENTED** [review]
Format companies.json

File: assets/data/companies.json (hunk 10 above)
Comment (harsh-mody) [reply assets/data/companies.json:3]: done

**Review by SirusCodes — APPROVED** [review]

**Pull request merged**